test/integration/infra/eks/providers.tf

```diff
@@ -6,12 +6,12 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 4.47.0"
+      version = "5.36.0"
```

Comment:
I think we'll have to update some of the terraform before moving to v5 here:

```shell-session

$ make -f aws.mk create-eks AWS_REGION=us-west-2
...
/opt/homebrew/bin/terraform -chdir=./test/integration/infra/eks/state apply -auto-approve \
		-var region=us-west-2 \
		-var kubernetes_version=1.26 || exit 1 \
	rm -f ./test/integration/infra/eks/state/*.tfvars
╷
│ Warning: Argument is deprecated
│ 
│   with module.vpc.aws_eip.nat,
│   on .terraform/modules/vpc/main.tf line 1013, in resource "aws_eip" "nat":
│ 1013:   vpc = true
│ 
│ use domain attribute instead
╵
╷
│ Error: Unsupported argument
│ 
│   on .terraform/modules/vpc/main.tf line 35, in resource "aws_vpc" "this":
│   35:   enable_classiclink             = null # https://github.com/hashicorp/terraform/issues/31730
│ 
│ An argument named "enable_classiclink" is not expected here.
╵
╷
│ Error: Unsupported argument
│ 
│   on .terraform/modules/vpc/main.tf line 36, in resource "aws_vpc" "this":
│   36:   enable_classiclink_dns_support = null # https://github.com/hashicorp/terraform/issues/31730
│ 
│ An argument named "enable_classiclink_dns_support" is not expected here.
╵
╷
│ Error: Unsupported argument
│ 
│   on .terraform/modules/vpc/main.tf line 1246, in resource "aws_default_vpc" "this":
│ 1246:   enable_classiclink   = null # https://github.com/hashicorp/terraform/issues/31730
│ 
│ An argument named "enable_classiclink" is not expected here.
╵
bash: line 0: exit: too many arguments
make: *** [create-eks] Error 1

```

Reply:
Should be good to go now with 97a1267a9fed977589a58f0565e409cafca546a3

```
irsa_role = "arn:aws:iam::xxxxx:role/iam-irsa-role-wuuv"
oidc_discovery_url = "https://oidc.eks.us-west-2.amazonaws.com/id/837BD2C2FDxxxxx0CF00212413253C"
region = "us-west-2"

github.com/hashicorp/vault-secrets-operator on  chore-bump-tf-provider-versions 13:08:42
➜  bash-QW63H4J7RD $ make -f aws.mk create-eks AWS_REGION=us-west-2
```

Reply:
Yep, seems to be working for me too 👍